src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/AnsiTerminalTestProgressFrame.cs

```diff
@@ -37,7 +37,7 @@ public void AppendTestWorkerProgress(TestProgressState progress, RenderedProgres
 
         terminal.Append('[');
         charsTaken++;
-        terminal.SetColor(TerminalColor.Green);
+        terminal.SetColor(TerminalColor.DarkGreen);
```

Comment:
Suggestion: What if we have `Constants.OurOwnGreen = TerminalColor.DarkGreen` and then use `OurOwnGreen` in all places. For some strange reason if we have to change DarkGreen to LightGreen, then we can simply change OurOwnGreen value,

Reply:
I hear you, but I would like to skip this for now. I don't see a reason why we would change the color. If we would unify to 1 color identifier, I would rather do it as a public option that sets the color for success, warning and error, and similarly progress / result indicators (like +, x, ?) for the same thing. There has been previous asks for distinguishing inner tests from outer tests in the output.